airmozilla/main/templates/main/event.html

```diff
@@ -139,75 +139,108 @@ <h2 class="event-date">
         </p>
       {% endif %}
       <p>{{ event.description | carefulnl2br | safe_html }}</p>
+      <div class="tabs">
 
+      {% if event.additional_links %}
+      <div class="tab">
+        <input type="radio" id="addlinks" name="tab-group-1">
```

Comment:
All of these `id="something"` can you change them to something like `id="tab-something"`.
Some names are too generic that it could interfere with other things. For example `id="tags"` is very generic.